ion/writer.go

```diff
@@ -166,3 +174,24 @@ func (w *writer) clear() {
 	w.fieldName = ""
 	w.annotations = nil
 }
+
+// Resolve resolves a symbol to its ID.
+func (w *writer) resolve(api, sym string) (uint64, error) {
```

Comment:
Could you add a comment to the function signature that explains what the `api` variable is for?